src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+
+import { ReactComponent as ArrowDropdown } from 'assets/images/icons/arrow-dropdown.svg';
+import { Answer } from 'types/answer';
+
+export const dropdownDataTestIds = {
+  base: 'dropdown__base',
+};
+
+interface DropdownProps {
+  questionId: string;
+  items: Answer[];
+  onValueChanged?: (value: Answer) => void;
+}
+const Dropdown = ({ questionId, items, onValueChanged }: DropdownProps): JSX.Element => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [selectedValue, setSelectedValue] = useState(items[0].text);
+
+  const toggleDropdown = () => {
+    setIsOpen((prev) => !prev);
+  };
+
+  const handleOnSelectValue = (value: Answer) => {
+    setSelectedValue(value.text);
+    onValueChanged?.(value);
+    toggleDropdown();
+  };
+
+  useEffect(() => {
+    setSelectedValue(items[0].text);
+  }, [questionId, items]);
+
+  return (
+    <div className="relative flex flex-col items-center w-full rounded-lg" data-test-id={dropdownDataTestIds.base}>
+      <button
+        onClick={() => toggleDropdown()}
+        className="w-full h-14 flex items-center justify-between appearance-none bg-white bg-opacity-[.18] rounded-[12px] hover:outline-none hover:ring-transparent px-3 text-white text-regular tracking-survey-tight hover:bg-opacity-30"
+      >
+        {selectedValue}
+        {!isOpen ? <ArrowDropdown /> : <ArrowDropdown className="rotate-180" />}
```

Comment:
```suggestion
        <ArrowDropdown className={isOpen ? '' : 'rotate-180'} />
```

Reply:
@nvminhtue fixed in [4803f51](https://github.com/manh-t/react-survey/pull/78/commits/4803f511251c858a44008f8f4f3580956f568ca8)